apps/web/src/app/[locale]/event/[eventCode]/CardList.tsx

```
@@ -0,0 +1,96 @@
+
+const slotContainerStyle = css({ position: 'relative', width: '265px', height: '328px', overflow: 'hidden' });
+
+const slotItemStyle = css({
+  position: 'absolute',
+  width: '100%',
+  height: '100%',
+});
+
+export function MobileCardList() {
+  return (
+    <Suspense fallback={<></>}>
```

Biome

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)